helper_functions.c

```diff
@@ -0,0 +1,80 @@
+#include "holberton.h"
+
+/**
+ * _strlen_recursion - length of string
+ * @s: The string
+ * Return: length of string
+ */
+int _strlen_recursion(char *s)
+{
+	if (*s == '\0')
+	{
+		return (0);
+	}
+	else
+		return (1 * _strlen_recursion(s + 1));
+}
+
+/**
+ * _atoi - convert a string s to an integer
+ * @s: The source array
+ *
+ * Return: The integer value or 0 if s isn't a number
+ */
+int _atoi(char *s)
```

Comment:
itoa function needed --> converts an integer to a character
[about itoa function](https://www.geeksforgeeks.org/implement-itoa/)